dependencies/bare-metal.yml

```diff
@@ -0,0 +1,12 @@
+name: test_purl_obolibrary
+channels:
+  - defaults
+  - conda-forge
+  - anaconda
```

Comment:
Why do we need Anaconda? Why use Python 2? The PURL system requires Python 3: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/Makefile#L16

Reply:
I guess this is for local testing on macOS?